packages/matrix/tests/messages.spec.ts

```diff
@@ -190,29 +169,6 @@ test.describe('Room messages', () => {
     );
   });
 
-  test('message sender has left room', async ({ page }) => {
```

Comment:
there is no longer anything special about this scenario now that we are using our room card so i removed this test